ubcemergingmedialab/3DMetabolism
# zoom component

# (brief description / PR title)

there is a camera attached per edge. You initially start in the main camera when running the program. When you click on an edge you will first be animated in the main camera's rig towards the edge after which the main camera will be disabled and you will be transferred into the edge's camera. Once you click the event plane(currently pink) you will switch back into the main camera which begins the zoom animation back out to your original position and rotation.

Status: **Ready for review!**

Trello: **https://trello.com/c/eBZmybLh**

## Changes
- implemented a zoom in component to zoom into the edges.

## How to test-drive this PR
- switch to multipleCamera-zoom branch and run the node on server.js
- Click any edge to zoom into the edge. You will know you are zoomed in when your screen changes to a pink color( this 
    pink plane lies inside the edge itself.
- click the anywhere on the red plane. You should be brought back to your initial position and rotation before zooming in
